rednotebook/external/text2rednotebook.py

```diff
@@ -0,0 +1,145 @@
+"""
+Mass import diary/journal entries from markdown/plain text/odt into rednotebook
+"""
+
+import argparse
+import sys
+import re
+import datetime
+from odf import text
+from odf.opendocument import load
```

Comment:
I think it would be great if the odf import was optional, so users who want to import plain text don't need to install it.

Reply:
Done